src/QsCompiler/BondSchemas/BondSchemaTranslator.cs

```diff
@@ -88,7 +88,7 @@ private static Identifier ToBondSchema(this SyntaxTree.Identifier identifier)
         private static LocalDeclarations ToBondSchema(this SyntaxTree.LocalDeclarations localDeclarations) =>
             new LocalDeclarations
             {
-                Variables = localDeclarations.Variables.Select(v => v.ToBondSchemaGeneric(typeTranslator: ToBondSchema)).ToList()
+                Variables = localDeclarations.Variables.Select(v => v.ToBondSchemaGeneric(typeTranslator: s => s)).ToList()
```

Comment:
It's intentional. The overload of `ToBondSchema` this line was using was:

```csharp
private static string ToBondSchema(this NonNullable<string> s) => s.Value;
```

which simply becomes the identity function since `NonNullable` is removed.